morgoth/base/utils.py

```diff
@@ -0,0 +1,70 @@
+import requests
+
+from datetime import datetime
+
+from django.conf import settings
+
+
+class BalrogAPI(object):
+    def __init__(self, auth=None):
+        """ Creates an object which wraps the Balrog REST API."""
+        self.auth = auth
+        self.session = requests.session()
+        self.csrf_token = None
+
+    def _is_csrf_token_expired(self):
+        if not self.csrf_token:
+            return True
+
+        expiry = self.csrf_token.split('##')[0]
+        if expiry <= datetime.now().strftime('%Y%m%d%H%M%S'):
```

Comment:
Although this works, thanks to the string format used, I think it would be better to parse expiry into a datetime. Something like

```python
return datetime.strftime(expiry, '%Y%m%d%H%M%S') <= datetime.now()
```